[Bug]: FileNotFoundException over resources/datahike-logo.txt

### What version of Datahike are you using?

0.6.1550

### What version of Java are you using?

19.0.2

### What operating system are you using?

Ubuntu 22.04.3 LTS

### What database EDN configuration are you using?

{:store {:backend :mem :id "in-mem-nippy-data"}
 :wanderung/type :datahike
 :schema-flexibility :write
 :attribute-refs? true
 :index :datahike.index/persistent-set
 :keep-history true
 :name "Nippy testing data"}

### Describe the bug

Trying to start the system generates an exception with the following part:
```
Caused by: java.io.FileNotFoundException: resources/datahike-logo.txt (No such file or directory)
        at java.base/java.io.FileInputStream.open0(Native Method)
        at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
```

### What is the expected behaviour?

There should not be a crash complaining about a missing resource file when trying to start a software using datahike.

### How can the behaviour be reproduced?

```
johan in 🌐 Labora in ~/Gits/daha via ☕ v19.0.2 on ☁️  (eu-central-1) took 4s
❯ bat deps.edn
───────┬─────────────────────────────────────────────────────────────────────────
       │ File: deps.edn
───────┼─────────────────────────────────────────────────────────────────────────
   1   │ {:deps {io.replikativ/datahike {:mvn/version "0.6.1550"}}
   2   │  :paths ["src"]}
───────┴─────────────────────────────────────────────────────────────────────────

johan in 🌐 Labora in ~/Gits/daha via ☕ v19.0.2 on ☁️  (eu-central-1)
❯ bat src/code.clj
───────┬─────────────────────────────────────────────────────────────────────────
       │ File: src/code.clj
───────┼─────────────────────────────────────────────────────────────────────────
   1   │ (ns code
   2   │   (:require [datahike.api :as dh]))
   3   │ (defn -main [& args]
   4   │   (print "Hello world"))
───────┴─────────────────────────────────────────────────────────────────────────
johan in 🌐 Labora in ~/Gits/daha via ☕ v19.0.2 on ☁️  (eu-central-1)
❯ clj -M -m code
Execution error (FileNotFoundException) at java.io.FileInputStream/open0 (FileInputStream.java:-2).
resources/datahike-logo.txt (No such file or directory)

Full report at:
/tmp/clojure-17817072141337129200.edn

```

--- a/config.edn
+++ b/config.edn
@@ -11,6 +11,7 @@
 
  :build {:clj {;; jvm build
                :src-dirs ["src"]
+               :resource-dir "resources"
                :java-src-dirs ["java"]
                :target-dir "target"
                :class-dir "target/classes"

--- a/deps.edn
+++ b/deps.edn
@@ -20,7 +20,7 @@
         org.babashka/http-client                    {:mvn/version "0.3.11"}
         metosin/jsonista                            {:mvn/version "0.3.7"}}
 
- :paths ["src" "target/classes"]
+ :paths ["src" "target/classes" "resources"]
 
  :deps/prep-lib {:ensure "target/classes"
                  :alias :build

--- a/src/datahike/tools.cljc
+++ b/src/datahike/tools.cljc
@@ -148,4 +148,4 @@
   #?(:clj (.getHostAddress (InetAddress/getLocalHost))
      :cljs (raise "Not supported yet." {:type :hostname-not-supported-yet})))
 
-(def datahike-logo (slurp "resources/datahike-logo.txt"))
+(def datahike-logo (slurp (io/resource "datahike-logo.txt")))

